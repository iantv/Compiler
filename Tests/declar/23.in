struct point{
	int x;
	int y;
};

struct point A;