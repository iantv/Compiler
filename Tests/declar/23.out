A: variable
	struct point
		x: variable
			int
		y: variable
			int
struct point
	x: variable
		int
	y: variable
		int
